specification/refarch.adoc

```diff
@@ -147,311 +189,315 @@ assigned to the TVM by the VMM.
 
 === TSM operation and properties
 
-The TSM implements security routines that are invoked by the OS/VMM or by 
-the TVMs, e.g. by the VMM to grant a TVM a TEE-capable memory page and 
-setup second-stage mapping, activate a TVM virtual hart on a physical hart 
-etc. The TSM security routines are invoked by the OS/VMM via an ECALL with 
-the service call specified via registers. These service calls trap to the 
-TSM-driver. The TSM-driver switches hart state to the TSM context by 
-loading the hart's TSM execution state from the THCS.tssa and then returns 
-via an MRET to the TSM. The TSM executes the security routine requested 
-(where the TSM enforces the security properties) and may either return to 
-the OS/VMM via an ECALL to the TSM-driver (TEERET with reason), or may use 
-an SRET to return/enter into a TVM. On a subsequent TVM synchronous or 
-asynchronous trap (due to ECALLs or any exception/interrupt) from a TVM, 
-the TSM handles the cases delegated to it by the TSM-driver (via mideleg). 
-The TSM saves the TVM state and invokes the TSM-driver via an ECALL (TEERET 
-with reason) to initiate the return of execution control to the OS/VMM if 
-required. The TSM-driver restores the context for the OS/VMM via the 
+The TSM implements security routines that are invoked by the OS/VMM or by
+the TVMs, e.g. by the VMM to grant a TVM a TEE-capable memory page and
+setup second-stage mapping, activate a TVM virtual hart on a physical hart
+etc. The TSM security routines are invoked by the OS/VMM via an ECALL with
+the service call specified via registers. These service calls trap to the
+TSM-driver. The TSM-driver switches hart state to the TSM context by
+loading the hart's TSM execution state from the THCS.tssa and then returns
+via an MRET to the TSM. The TSM executes the security routine requested
+(where the TSM enforces the security properties) and may either return to
+the OS/VMM via an ECALL to the TSM-driver (TEERET with reason), or may use
```

Comment:
```suggestion
the hosting supervisor domain manager OS/VMM via an ECALL to the TSM-driver (TEERET with reason), or may use
```

In general, should we replace the `OS/VMM` instances with either `hosting supervisor domain manager` or `hosting supervisor domain manager OS/VMM` ?

Reply:
may be too verbose - will mention it at the intro and then stick with OS/VMM.